.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: rust
-cache: cargo
+cache: 
+  cargo: true
```

Comment:
More concretely, if you change this to `false` but keep the `cargo_web` cache enabled, maybe it's already good enough to make the Windows builds fast (I don't think so.)  If it's not enough, you can add the `before_cache` phase and let it remove the contents of the `cargo_web` directory only on windows. As Travis keeps separate caches for Windows and Linux, this won't affect the Linux builds where the web cache is used.

Reply:
right now `cargo web` isn't used in windows at all. so removing the `cargo` cache will leave windows with no cache at all (which is fine I guess)

Reply:
Right, that was my suggestion because the cache makes it slower on windows.